src/checkpoint.hh

```diff
@@ -1282,9 +1521,10 @@ SUPPORTED_EXTERNAL_TYPES = {"Rope", "answer_pknot_mfe", "pktype",
   }
 
   void parse_checkpoint_log(Printer::Base &stream) {
-     inc_indent(); inc_indent();
+     inc_indent();
+     if (!cyk) inc_indent();
      stream << indent() << "void parse_checkpoint_log("
```

Comment:
identation in generated code seems to be off by one

Reply:
Fixed.